src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7418,4 +7418,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ParamsArrayInLambdaOnly_Title" xml:space="preserve">
     <value>Parameter has params modifier in lambda but not in target delegate type.</value>
   </data>
+  <data name="IDS_DisableLengthBasedSwitch" xml:space="preserve">
+    <value>disable-length-based-switch</value>
```

Comment:
>disable-length-based-switch [](http://example.com/codeflow?start=11&length=27)

Is this string intended to be localized?

Reply:
We won't need it localized.
It's the same as other `IDS_` entries above, such as `IDS_FeatureLambdaParamsArray`.
Will add a comment